entities/Order.java

```diff
@@ -0,0 +1,68 @@
+package entities;
+
+import constants.Status;
+
+import java.time.Instant;
+import java.util.Date;
+import java.util.List;
+
+public class Order extends Base {
+    //Attributes
+
+    private String customerName;
+    private List<Product> items;
+    private Double totalAmount;
+    private Date orderDate;
```

Comment:
Use `Instant` instead

Reply:
Done. order Date removed cuz there is an attribute to this in the base entity